client/src/GlobalStyle.js

```diff
@@ -0,0 +1,58 @@
+import { createGlobalStyle } from "styled-components/macro";
+import AtamiRegular from "./stories/assets/fonts/AtamiRegular.otf";
+import AtamiBold from "../src/stories/assets/fonts/AtamiBold.otf";
+
+const GlobalStyle = createGlobalStyle`
+ *, *::after, *::before {
+   box-sizing:border-box;
+ }
+
+
+ @font-face {
+  font-family: "AtamiRegular";
+  font-style: normal;
+  font-weight: 400;
+  src: url(${AtamiRegular});
+ }
+
+  @font-face {
+  font-family: "AtamiBold";
+  font-style: bold;
+  font-weight: 600;
+  src: url(${AtamiBold})
+}
+
+html {
+  height: 100%;
+}
+ 
+ :root {
+   --primary-color: #fbfbfb;
+   --guidance-color: #0a335c;
+   --decision-primary-color: #ff4477;
+   --decision-1-color: #ffd17e;
```

Comment:
```suggestion
   --decision-secondary-color: #ffd17e;
```

maybe rename that number in that variable...probably decision-secondary-color ?